code/datums/mutations/body.dm

```diff
@@ -514,3 +514,39 @@
 
 	if(istype(new_limb, /obj/item/bodypart/head))
 		return COMPONENT_NO_ATTACH
+
+/datum/mutation/human/tiny
+	name = "Tiny"
+	desc = "The cells in the host conregate together, causing the host to appear smaller."
+	quality = NEGATIVE
+	difficulty = 16
+	instability = 5
+	conflicts = list(GIGANTISM, DWARFISM)
+	locked = TRUE
```

Comment:
do you, uh, have a mutation combination recipe for this yet?

Reply:
or is it supposed to be admin-only?

Reply:
> do you, uh, have a mutation combination recipe for this yet?

I don't think it needs one. what would it even be?

Reply:
Dwarfism and some other compoment mutation, such as maybe Void Magnet or Spatial Instability.